packages/client/src/rpc/modules/engine.ts

```diff
@@ -176,7 +177,11 @@ export const blockToExecutionPayload = (block: Block, value: bigint, bundle?: Bl
     transactions,
     ...withdrawalsArr,
   }
-  return { executionPayload, blockValue: bigIntToHex(value), blobsBundle }
+
+  // ethereumjs doesnot provide any transaction censoring detection (yet) to suggest
+  // overriding builder/mev-boost blocks
+  const shouldOverrideBuilder = false
+  return { executionPayload, blockValue: bigIntToHex(value), blobsBundle, shouldOverrideBuilder }
```

Comment:
see the comment on the top:

since we don't have any transaction censoring detection features,, we just return it as false (which is what is recommended to be returned if an EL doesn't has such features so the CL can choose builder/mev-boost block if it wants to)